rtf_vis_tool/src/Components/AllFrustums.js

```diff
@@ -0,0 +1,119 @@
+/* Component that renders all of the Frustums using the files: cameras.txt and images.txt. Will be rendered within
+Bundle_Adj_PC.js.
+
+Author: Adi Singh
+*/
+import React, {useEffect, useState} from "react";
+import Frustum from './Frustum';
+
+// Third-Party Package Imports.
+var nj = require('numjs');
+var Quaternion = require('quaternion');
+
+// Local Imports.
+var SE3 = require('./frustum_classes/se3');
+var ViewFrustum = require('./frustum_classes/view_frustum');
+
+function AllFrustums() {
+    /* 
+    Returns:
+        A component rendering all the camera frustums.
+    */
+    const [frustumsJSX, setFrustumsJSX] = useState([]);
+
+    // Load camera intrinsics and extrinsics by fetching cameras.txt and images.txt respectively.
+    useEffect(() => {
```

Comment:
is this the non-standard (by my book :-) ) function syntax here for `useEffect()`, or is it required here?

Reply:
So this syntax is specific to React. Unfortunately, I've only ever seen it written in this format and all the React documentation writes it like this, so I don't think we can change it. FYI, this is called a React Hook, which basically runs the code within once the component is first loaded.